renter/download.go

```diff
@@ -14,7 +14,7 @@ import (
 
 // ErrBadChecksum indicates that a piece of sector data failed checksum
 // validation.
-var ErrBadChecksum = errors.New("sector data failed checksum validation")
+var ErrBadChecksum = fmt.Errorf("sector data failed checksum validation")
```

Comment:
my preference is to stick with `errors.New` for simple string errors. It adds an extra import, but imo it communicates intent a bit more clearly.

Reply:
By intent, do you mean the fact that this is an error cannot be handled internally? If so, does the fact that it is exported not imply that?

Reply:
nah, I mean when I'm reading someone's code and see `errors.New` I think "okay, this is just an opaque string", but when I see `fmt.Errorf` I think "oh, this must be a fancier error that requires formatting or wrapping."

Reply:
I guess somewhere down the line I joined the printf-forever team and it corrupted my brain. That makes much more sense.